solr/test-framework/src/java/org/apache/solr/cloud/AbstractFullDistribZkTestBase.java

```diff
@@ -2225,9 +2225,7 @@ protected String getBaseUrl(HttpSolrClient client) {
   }
 
   public static SolrInputDocument getDoc(Object... fields) throws Exception {
-    SolrInputDocument doc = new SolrInputDocument();
-    addFields(doc, fields);
-    return doc;
+    return sdoc(fields);
   }
```

Comment:
outside scope in this PR here but perhaps long-term only one of `getDoc` and `sdoc` could be kept then.